### PR TITLE
Parse IPMSI parameter in cmd

### PR DESCRIPTION
**Pull request by JieJhih**
Ref. https://www.ietf.org/id/draft-ietf-bess-evpn-bum-procedure-updates-05.txt
```

   The per-region I-PMSI route uses an embedded EC in NLRI to identify a
   region.  As long as it uniquely identifies the region and the RBRs
   for the same region uses the same EC it is permitted.  In the case
   where an AS number or area ID is needed, the following can be used:

   o  For a two-octet AS number, a Transitive Two-Octet AS-Specific EC
      of sub-type 0x09 (Source AS), with the Global Administrator sub-
      field set to the AS number and the Local Administrator sub-field
      set to 0.

   o  For a four-octet AS number, a Transitive Four-Octet AS-Specific EC
      of sub-type 0x09 (Source AS), with the Global Administrator sub-
      field set to the AS number and the Local Administrator sub-field
      set to 0.
```

### CODE REVIEW

File: pkg/packet/bgp/bgp.go [hunk 1]
```diff
@@ -10029,8 +10032,8 @@ func ParseExtendedCommunity(subtype ExtendedCommunityAttrSubType, com string) (E
 	}
 }
 
-func ParseRouteTarget(rt string) (ExtendedCommunityInterface, error) {
```
Comment (fujita) [inline pkg/packet/bgp/bgp.go]: This function is publicly exported. You can't change it without the major version updated.

Comment (JieJhih) [reply pkg/packet/bgp/bgp.go]: Fixed.
Thanks

**Comment by fujita** [comment]
Thanks, now the code looks good. Can you update the doc?
https://github.com/osrg/gobgp/blob/master/docs/sources/evpn.md#cli-syntax

**Comment by JieJhih** [comment]
> Thanks, now the code looks good. Can you update the doc?

Updated.
Thanks

**Comment by fujita** [comment]
squashed and pushed, thanks.

**Pull request closed without merging**